crates/sui-config/src/node.rs

```diff
@@ -412,6 +412,14 @@ pub struct ConsensusConfig {
     // Base consensus DB path for all epochs.
     pub db_path: PathBuf,
 
+    // The number of epochs for which to retain the consensus DBs. Setting it to 0 will make an epoch DB getting
```

Comment:
are consensus DBs and epoch DBs equivalent terms?

Reply:
I'll stick to consensus dbs